src/nrniv/nrnbbcore_write/data/cell_group.h

```diff
@@ -0,0 +1,78 @@
+#ifndef NRN_CELL_GROUP_H
+#define NRN_CELL_GROUP_H
+
+#include "datum_indices.h"
+#include <vector>
+#include <map>
+
+struct Memb_list;
+class PreSyn;
+class NetCon;
+class NrnThread;
+
+extern "C" {
+
+    extern int nrn_dblpntr2nrncore(double*, NrnThread&, int& type, int& etype);
+
+}
+
+typedef std::pair < int, Memb_list* > MlWithArtItem;
+typedef std::vector < MlWithArtItem > MlWithArt;
+typedef std::map<double*, int> PVoid2Int;
```

Comment:
Used an `std::map` here instead of `declareNrnHash` because there was no specific extra usage.